upstream-community-operators/rook-ceph/rook-ceph.v1.0.1.clusterserviceversion.yaml

```diff
@@ -444,7 +444,7 @@ spec:
   - email: info@rook.io
     name: The Rook Authors
   maturity: alpha
-  replaces: rook-ceph.v1.0.0
+  replaces: rook-ceph.v1.0.1
```

Comment:
This must be causing the build failure.  You are referring to same CSV version as a replacement.
The replace is used for upgrades so OLM knows it is to replace the older version, not the same version.  If not replacing the older version, then  remove the replace metadata

Reply:
@leseb fixing the replaces will progress the build.

Reply:
Yup that's what I did.